docs/README.md

```diff
@@ -100,6 +100,23 @@ ld -T kernel.ld -o kernel.bin kernel.o
 The flags added to the first command are there to help GCC produce could that will run in kernel space.
 The second command simply takes kernel.o and orders it as the linker script tells it to.
 
+## Contributors note
+
+After a bit of testing I found out that the entry symbol (defaults to \_start) must always be in front of anything - even includes. This is probably due to Pure64 loading pure binaries which don't store the entry points address. 
+The best solutions I could come up with is to create a seperate file (e.g `start.c`) that calls the `main` function.
```

Comment:
Remove first person form:

_The best solution is to put the entry piont in a separate file that calls the `main` function._

This advice might change in the future if we bring back newlib with a CRT file, but is good advice for right now.